diff --git a/assets/js/login.js b/assets/js/login.js
index 63c74cc7..d41d74e8 100644
--- a/assets/js/login.js
+++ b/assets/js/login.js
@@ -22,7 +22,7 @@ $(document).ready(function() {
 
         if ($usernameInput.val().length >= 20) {
             require.ensure(['./Components/show_long_username_error'], (require) => {
-                const showLongUsernameError = require('./Components/show_long_username_error');
+                const showLongUsernameError = require('./Components/show_long_username_error').default;
                 showLongUsernameError($usernameInput);
             });
         }
