diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index 4c03dad6..3526e289 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -9,8 +9,6 @@
     <title>{% block title %}Find Something to Lift!{% endblock %}</title>
 
     {% block stylesheets %}
-        <link href="https://netdna.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" />
-        <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous" />
         <link href="{{ asset('build/layout.css') }}" rel="stylesheet" />
     {% endblock %}
 
diff --git a/assets/js/layout.js b/assets/js/layout.js
index b5e2feb6..3bf1b222 100644
--- a/assets/js/layout.js
+++ b/assets/js/layout.js
@@ -2,6 +2,8 @@
 
 const $ = require('jquery');
 require('bootstrap');
+require('bootstrap/dist/css/bootstrap.css');
+require('font-awesome/css/font-awesome.css');
 require('../css/layout.css');
 
 $(document).ready(function() {
diff --git a/package.json b/package.json
index 83b6cb0d..013d7b1f 100644
--- a/package.json
+++ b/package.json
@@ -3,6 +3,7 @@
     "@symfony/webpack-encore": "^0.17.0",
     "bootstrap": "^3.3.7",
     "copy-webpack-plugin": "^4.2.0",
+    "font-awesome": "^4.7.0",
     "jquery": "^3.2.1",
     "webpack-notifier": "^1.5.0"
   }
diff --git a/yarn.lock b/yarn.lock
index d8f2aecd..892bfd93 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -1946,6 +1946,10 @@ flatten@^1.0.2:
   version "1.0.2"
   resolved "https://registry.yarnpkg.com/flatten/-/flatten-1.0.2.tgz#dae46a9d78fbe25292258cc1e780a41d95c03782"
 
+font-awesome@^4.7.0:
+  version "4.7.0"
+  resolved "https://registry.yarnpkg.com/font-awesome/-/font-awesome-4.7.0.tgz#8fa8cf0411a1a31afd07b06d2902bb9fc815a133"
+
 for-in@^1.0.1:
   version "1.0.2"
   resolved "https://registry.yarnpkg.com/for-in/-/for-in-1.0.2.tgz#81068d295a8142ec0ac726c6e2200c30fb6d5e80"
