diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 585d1bf5..d2a4c7f7 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -56,11 +56,4 @@
 
     <script src="https://cdn.jsdelivr.net/sweetalert2/6.1.0/sweetalert2.min.js"></script>
     <script src="{{ asset('build/rep_log.js') }}"></script>
-
-    <script>
-        $(document).ready(function() {
-            var $wrapper = $('.js-rep-log-table');
-            var repLogApp = new RepLogApp($wrapper);
-        });
-    </script>
 {% endblock %}
diff --git a/web/assets/js/rep_log.js b/web/assets/js/rep_log.js
new file mode 100644
index 00000000..a42e5e7f
--- /dev/null
+++ b/web/assets/js/rep_log.js
@@ -0,0 +1,6 @@
+const RepLogApp = require('./RepLogApp');
+
+$(document).ready(function() {
+    var $wrapper = $('.js-rep-log-table');
+    var repLogApp = new RepLogApp($wrapper);
+});
diff --git a/webpack.config.js b/webpack.config.js
index 2f3fc9f9..b4723e5e 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -12,7 +12,7 @@ Encore
     .cleanupOutputBeforeBuild()
 
     // will output as web/build/app.js
-    .addEntry('rep_log', './web/assets/js/RepLogApp.js')
+    .addEntry('rep_log', './web/assets/js/rep_log.js')
 
     // allow sass/scss files to be processed
     //.enableSassLoader()
