diff --git a/app/Resources/views/base.html.twig b/app/Resources/views/base.html.twig
index ccba9c2f..378e47d3 100644
--- a/app/Resources/views/base.html.twig
+++ b/app/Resources/views/base.html.twig
@@ -100,6 +100,7 @@
     <script src="{{ asset('bundles/fosjsrouting/js/router.js') }}"></script>
     <script src="{{ path('fos_js_routing_js', { callback: 'fos.Router.setData' }) }}"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/es6-promise/4.0.5/es6-promise.auto.min.js"></script>
+    <script src="{{ asset('build/manifest.js') }}"></script>
     <script src="{{ asset('build/layout.js') }}"></script>
 {% endblock %}
 
diff --git a/webpack.config.js b/webpack.config.js
index cda3d6fc..ab1e5374 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -15,7 +15,7 @@ Encore
     // will output as web/build/app.js
     .addEntry('rep_log', './web/assets/js/rep_log.js')
     .addEntry('login', './web/assets/js/login.js')
-    .addEntry('layout', './web/assets/js/layout.js')
+    .createSharedEntry('layout', './web/assets/js/layout.js')
 
     // allow sass/scss files to be processed
     //.enableSassLoader()
