diff --git a/web/assets/css/layout.css b/assets/css/layout.css
similarity index 100%
rename from web/assets/css/layout.css
rename to assets/css/layout.css
diff --git a/web/assets/css/login.css b/assets/css/login.css
similarity index 100%
rename from web/assets/css/login.css
rename to assets/css/login.css
diff --git a/web/assets/images/dumbbell-mini.png b/assets/images/dumbbell-mini.png
similarity index 100%
rename from web/assets/images/dumbbell-mini.png
rename to assets/images/dumbbell-mini.png
diff --git a/web/assets/images/dumbbell.png b/assets/images/dumbbell.png
similarity index 100%
rename from web/assets/images/dumbbell.png
rename to assets/images/dumbbell.png
diff --git a/web/assets/js/Components/RepLogApp.js b/assets/js/Components/RepLogApp.js
similarity index 100%
rename from web/assets/js/Components/RepLogApp.js
rename to assets/js/Components/RepLogApp.js
diff --git a/web/assets/js/Components/RepLogAppHelper.js b/assets/js/Components/RepLogAppHelper.js
similarity index 100%
rename from web/assets/js/Components/RepLogAppHelper.js
rename to assets/js/Components/RepLogAppHelper.js
diff --git a/web/assets/js/layout.js b/assets/js/layout.js
similarity index 100%
rename from web/assets/js/layout.js
rename to assets/js/layout.js
diff --git a/web/assets/js/login.js b/assets/js/login.js
similarity index 100%
rename from web/assets/js/login.js
rename to assets/js/login.js
diff --git a/web/assets/js/rep_log.js b/assets/js/rep_log.js
similarity index 100%
rename from web/assets/js/rep_log.js
rename to assets/js/rep_log.js
diff --git a/webpack.config.js b/webpack.config.js
index ab1e5374..0603fad0 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -13,9 +13,9 @@ Encore
     .enableBuildNotifications()
 
     // will output as web/build/app.js
-    .addEntry('rep_log', './web/assets/js/rep_log.js')
-    .addEntry('login', './web/assets/js/login.js')
-    .createSharedEntry('layout', './web/assets/js/layout.js')
+    .addEntry('rep_log', './assets/js/rep_log.js')
+    .addEntry('login', './assets/js/login.js')
+    .createSharedEntry('layout', './assets/js/layout.js')
 
     // allow sass/scss files to be processed
     //.enableSassLoader()
