diff --git a/conference.js b/conference.js
index 088e7fe0..43a44a2c 100644
--- a/conference.js
+++ b/conference.js
@@ -2,10 +2,10 @@
 
 module.exports = class {
     constructor(name) {
-        this.name = name;
+        this._name = name;
     }
 
-    getName() {
-        return this.name;
+    get name() {
+        return this._name;
     }
 };
diff --git a/play.js b/play.js
index 8389802a..23c12504 100644
--- a/play.js
+++ b/play.js
@@ -2,4 +2,4 @@ const Conference = require('./conference');
 
 const sfCon = new Conference('SymfonyCon 2017');
 
-console.log(sfCon.getName());
+console.log(sfCon.name);
