diff --git a/assets/js/layout.js b/assets/js/layout.js
index b09d7585..5677d563 100644
--- a/assets/js/layout.js
+++ b/assets/js/layout.js
@@ -5,7 +5,7 @@ import 'bootstrap';
 import 'bootstrap/dist/css/bootstrap.css';
 import 'font-awesome/css/font-awesome.css';
 import '../css/layout.scss';
-import 'core-js';
+import 'core-js/fn/promise';
 
 $(document).ready(function() {
     $('[data-toggle="tooltip"]').tooltip();
