diff --git a/app/Resources/FOSUserBundle/views/Security/login.html.twig b/app/Resources/FOSUserBundle/views/Security/login.html.twig
index c7ecc471..8b776b08 100644
--- a/app/Resources/FOSUserBundle/views/Security/login.html.twig
+++ b/app/Resources/FOSUserBundle/views/Security/login.html.twig
@@ -19,6 +19,7 @@
         <div class="wrapper">
             <form action="{{ path("fos_user_security_check") }}" method="post" class="form-signin">
                 <h3><img class="dumbbell" src="{{ asset('build/static/dumbbell.png') }}">Login! Start Lifting!</h3>
+                <div id="js-login-details"></div>
                 <div class="js-recommended-login">
                     <a href="#" class="js-show-login">Don't know the login?</a>
                     <div class="js-recommended-login-details" style="display: none;">
diff --git a/assets/js/login.js b/assets/js/login.js
index c9c49053..89f83ebe 100644
--- a/assets/js/login.js
+++ b/assets/js/login.js
@@ -3,8 +3,15 @@
 import $ from 'jquery';
 import '../css/login.css';
 import LoginDetails from './Components/LoginDetails';
+import ReactDOM from 'react-dom';
+import React from 'react';
 
 $(document).ready(function() {
+    ReactDOM.render(
+        <LoginDetails />,
+        document.getElementById('js-login-details')
+    );
+
     $('.js-recommended-login').on('click', '.js-show-login', function(e) {
         e.preventDefault();
 
