diff --git a/app/Resources/FOSUserBundle/views/Security/login.html.twig b/app/Resources/FOSUserBundle/views/Security/login.html.twig
index 8b776b08..ec19d84f 100644
--- a/app/Resources/FOSUserBundle/views/Security/login.html.twig
+++ b/app/Resources/FOSUserBundle/views/Security/login.html.twig
@@ -20,24 +20,6 @@
             <form action="{{ path("fos_user_security_check") }}" method="post" class="form-signin">
                 <h3><img class="dumbbell" src="{{ asset('build/static/dumbbell.png') }}">Login! Start Lifting!</h3>
                 <div id="js-login-details"></div>
-                <div class="js-recommended-login">
-                    <a href="#" class="js-show-login">Don't know the login?</a>
-                    <div class="js-recommended-login-details" style="display: none;">
-                        Sir or madam, might I recommend that you try the following login details:
-                        <table class="table">
-                            <tbody>
-                                <tr>
-                                    <th>Username</th>
-                                    <td>cindy_clawford</td>
-                                </tr>
-                                <tr>
-                                    <th>Password</th>
-                                    <td>pumpup</td>
-                                </tr>
-                            </tbody>
-                        </table>
-                    </div>
-                </div>
                 <hr class="colorgraph" />
 
                 <br/>
diff --git a/assets/js/Components/LoginDetails.jsx b/assets/js/Components/LoginDetails.jsx
index d15391cc..a9f1735f 100644
--- a/assets/js/Components/LoginDetails.jsx
+++ b/assets/js/Components/LoginDetails.jsx
@@ -3,8 +3,23 @@ import React from 'react';
 export default class LoginDetails extends React.Component {
     render() {
         return (
-            <div>
-                I am the login details!
+            <div class="js-recommended-login">
+                <a href="#" class="js-show-login">Don't know the login?</a>
+                <div class="js-recommended-login-details" style="display: none;">
+                    Sir or madam, might I recommend that you try the following login details:
+                    <table class="table">
+                        <tbody>
+                            <tr>
+                                <th>Username</th>
+                                <td>cindy_clawford</td>
+                            </tr>
+                            <tr>
+                                <th>Password</th>
+                                <td>pumpup</td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </div>
             </div>
         );
     }
diff --git a/assets/js/login.js b/assets/js/login.js
index 89f83ebe..7fa228ff 100644
--- a/assets/js/login.js
+++ b/assets/js/login.js
@@ -12,12 +12,6 @@ $(document).ready(function() {
         document.getElementById('js-login-details')
     );
 
-    $('.js-recommended-login').on('click', '.js-show-login', function(e) {
-        e.preventDefault();
-
-        $('.js-recommended-login-details').toggle();
-    });
-
     $('.js-login-field-username').on('keydown', function(e) {
         const $usernameInput = $(e.currentTarget);
         // remove any existing warnings
