diff --git a/app/Resources/FOSUserBundle/views/Security/login.html.twig b/app/Resources/FOSUserBundle/views/Security/login.html.twig
index ec19d84f..eb244c9b 100644
--- a/app/Resources/FOSUserBundle/views/Security/login.html.twig
+++ b/app/Resources/FOSUserBundle/views/Security/login.html.twig
@@ -19,7 +19,7 @@
         <div class="wrapper">
             <form action="{{ path("fos_user_security_check") }}" method="post" class="form-signin">
                 <h3><img class="dumbbell" src="{{ asset('build/static/dumbbell.png') }}">Login! Start Lifting!</h3>
-                <div id="js-login-details"></div>
+                <div id="js-login-details" data-username="cindy_clawford" data-password="pumpup"></div>
                 <hr class="colorgraph" />
 
                 <br/>
diff --git a/assets/js/login.js b/assets/js/login.js
index c22d4d6f..f1862f29 100644
--- a/assets/js/login.js
+++ b/assets/js/login.js
@@ -7,12 +7,12 @@ import ReactDOM from 'react-dom';
 import React from 'react';
 
 $(document).ready(function() {
-    const username = 'cindy_clawford';
-    const password = 'pumpup';
-
+    const loginDetailsEl = document.getElementById('js-login-details');
+    const username = loginDetailsEl.dataset.username;
+    const password = loginDetailsEl.dataset.password;
     ReactDOM.render(
         <LoginDetails username={username} password={password} />,
-        document.getElementById('js-login-details')
+        loginDetailsEl
     );
 
     $('.js-login-field-username').on('keydown', function(e) {
