diff --git a/assets/js/Components/LoginDetails.jsx b/assets/js/Components/LoginDetails.jsx
index a766d8d8..c70ef9fe 100644
--- a/assets/js/Components/LoginDetails.jsx
+++ b/assets/js/Components/LoginDetails.jsx
@@ -30,11 +30,11 @@ export default class LoginDetails extends React.Component {
                         <tbody>
                             <tr>
                                 <th>Username</th>
-                                <td>cindy_clawford</td>
+                                <td>{this.props.username}</td>
                             </tr>
                             <tr>
                                 <th>Password</th>
-                                <td>pumpup</td>
+                                <td>{this.props.password}</td>
                             </tr>
                         </tbody>
                     </table>
diff --git a/assets/js/login.js b/assets/js/login.js
index 7fa228ff..c22d4d6f 100644
--- a/assets/js/login.js
+++ b/assets/js/login.js
@@ -7,8 +7,11 @@ import ReactDOM from 'react-dom';
 import React from 'react';
 
 $(document).ready(function() {
+    const username = 'cindy_clawford';
+    const password = 'pumpup';
+
     ReactDOM.render(
-        <LoginDetails />,
+        <LoginDetails username={username} password={password} />,
         document.getElementById('js-login-details')
     );
 
