diff --git a/assets/js/Components/LoginDetails.jsx b/assets/js/Components/LoginDetails.jsx
index d15391cc..9fcf0ea9 100644
--- a/assets/js/Components/LoginDetails.jsx
+++ b/assets/js/Components/LoginDetails.jsx
@@ -2,9 +2,11 @@ import React from 'react';
 
 export default class LoginDetails extends React.Component {
     render() {
+        const name = 'Ryan';
+        
         return (
             <div>
-                I am the login details!
+                I am the login details! {name}
             </div>
         );
     }
