diff --git a/assets/js/Components/LoginDetails.jsx b/assets/js/Components/LoginDetails.jsx
index 5cf498cd..a766d8d8 100644
--- a/assets/js/Components/LoginDetails.jsx
+++ b/assets/js/Components/LoginDetails.jsx
@@ -8,6 +8,8 @@ export default class LoginDetails extends React.Component {
         this.state = {
             isVisible: false
         }
+
+        this.onToggleClick = this.onToggleClick.bind(this);
     }
 
     onToggleClick(e) {
