diff --git a/assets/js/Components/LoginDetails.jsx b/assets/js/Components/LoginDetails.jsx
index 8e1b4454..b1b7f763 100644
--- a/assets/js/Components/LoginDetails.jsx
+++ b/assets/js/Components/LoginDetails.jsx
@@ -1,11 +1,20 @@
 import React from 'react';
 
 export default class LoginDetails extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isVisible: false
+        }
+    }
+
     render() {
         return (
             <div>
                 <a href="#" >Don't know the login?</a>
-                <div style={{display: 'none'}}>
+                <div className={this.state.isVisible ? '' : 'hidden'}>
                     Sir or madam, might I recommend that you try the following login details:
                     <table className="table">
                         <tbody>
