diff --git a/.gitignore b/.gitignore
index 0654fa26..e42d59f8 100644
--- a/.gitignore
+++ b/.gitignore
@@ -15,3 +15,4 @@
 /vendor/
 /web/bundles/
 /node_modules
+/web/build
diff --git a/app/Resources/views/lift/index.html.twig b/app/Resources/views/lift/index.html.twig
index 739833b3..585d1bf5 100644
--- a/app/Resources/views/lift/index.html.twig
+++ b/app/Resources/views/lift/index.html.twig
@@ -55,7 +55,7 @@
     {{ parent() }}
 
     <script src="https://cdn.jsdelivr.net/sweetalert2/6.1.0/sweetalert2.min.js"></script>
-    <script src="{{ asset('assets/js/RepLogApp.js') }}"></script>
+    <script src="{{ asset('build/rep_log.js') }}"></script>
 
     <script>
         $(document).ready(function() {
diff --git a/webpack.config.js b/webpack.config.js
index 6412beea..2f3fc9f9 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -12,16 +12,13 @@ Encore
     .cleanupOutputBeforeBuild()
 
     // will output as web/build/app.js
-    .addEntry('app', './assets/js/main.js')
-
-    // will output as web/build/global.css
-    .addStyleEntry('global', './assets/css/global.scss')
+    .addEntry('rep_log', './web/assets/js/RepLogApp.js')
 
     // allow sass/scss files to be processed
-    .enableSassLoader()
+    //.enableSassLoader()
 
     // allow legacy applications to use $/jQuery as a global variable
-    .autoProvidejQuery()
+    //.autoProvidejQuery()
 
     .enableSourceMaps(!Encore.isProduction())
 
